Index: /youtubei.js/dist/src/core/Player.js
===================================================================
--- /youtubei.js/dist/src/core/Player.js
+++ /youtubei.js/dist/src/core/Player.js
@@ -17,9 +17,10 @@
         const res = await fetch(url);
         if (!res.ok)
             throw new PlayerError(`Failed to get player id: ${res.status} (${res.statusText})`);
         const js = await res.text();
-        const player_id = getStringBetweenStrings(js, 'player\\/', '\\/');
+        // const player_id = getStringBetweenStrings(js, 'player\\/', '\\/');
+        const player_id = "643afba4";
         Log.info(TAG, `Got player id (${player_id}). Checking for cached players..`);
         if (!player_id)
             throw new PlayerError('Failed to get player id');
         // We have the player id, now we can check if we have a cached player.
@@ -220,5 +221,5 @@
     static get LIBRARY_VERSION() {
         return 14;
     }
 }
-//# sourceMappingURL=Player.js.map
\ No newline at end of file
+//# sourceMappingURL=Player.js.map
